gno.land/cmd/gnoland/root.go

```diff
@@ -2,32 +2,49 @@ package main
 
 import (
 	"context"
+	"flag"
 	"os"
 
 	"github.com/gnolang/gno/tm2/pkg/commands"
 	"github.com/peterbourgon/ff/v3"
 	"github.com/peterbourgon/ff/v3/fftoml"
 )
 
-const flagConfigFlag = "flag-config-path"
-
 func main() {
 	cmd := newRootCmd(commands.NewDefaultIO())
 
 	cmd.Execute(context.Background(), os.Args[1:])
 }
 
+type rootCfg struct {
+	homeDir homeDirectory
+}
```

Comment:
please, for flag structs, don't nest them. They can have a helper type like the one you created, but there shouldn't be 3 levels of nesting

Reply:
I'm not sure to understand how you want it not nested ?

Reply:
Yeah, I don't know, I had a bit of a think and discussion with Milos and I'm not sure how this should work. cc @zivkovicmilos

I don't like `rootCfg` :shrug: